sw/tools/parrot/disco.py

```diff
@@ -38,7 +38,7 @@ def uav_status(self):
     def init_extra_parser(self):
 
         # Parse the extra arguments
-        self.parser.add_argument('--min_version', metavar='MIN', default='1.4.1',
+        self.parser.add_argument('--min_version', metavar='MIN', default='1.7.1',
```

Comment:
why rising the min version ? is it really not compatible anymore ?

Reply:
It is, as far as our last test compatible for upload but we do never fly anymore with a lower v than 1.7.1 The reason is if we want to use the Hardware encoding is could cause .ko/.so entrypoint issues again, And since Parrot stopped making the Disco v1.7.1 will be the last firmware ever. All Discos we test on (8) have v1.7.1. If there would be a benefit I'm fine with 1.4.1 but have no plan to test it since there is no benefit ATM and will heighten the risk someting goes wrong. Where it boiled down to was we want to limit the amount of firmware we need to testfly on just to wards one perfect flying Disco AC

Reply:
Increasing min version don't allow old airframe that might not have been updated to fly. I would not change that unless necessary because it is just not necessary. I don't think we need to test the older firmware either. If someone report an issue, we might increase the value later instead of supporting something not produced anymore.

Reply:
I understand, so fine byme I'll leave to v1.4.1 as min, not however we will only testfly v1.7.1 thus we cannot guarantee v1.4.1 still works 100% with PPRZ Master then. But it will not crash for us either then because of that ;)